plugins/base/src/main/kotlin/renderers/html/HtmlRenderer.kt

```diff
@@ -982,8 +987,5 @@ private val PageNode.isNavigable: Boolean
 private fun PropertyContainer<ContentNode>.extraHtmlAttributes() = allOfType<SimpleAttr>()
 private fun PropertyContainer<ContentNode>.extraTabbedContentType() = this[TabbedContentTypeExtra]
 
-private val ContentNode.sourceSetsFilters: String
-    get() = sourceSets.sourceSetIDs.joinToString(" ") { it.toString() }
```

Comment:
Before every table-row, there's an invisible link - I think it's used by the anchor link to focus on the right row. This property was used for the filter value in it, but I couldn't understand if it was actually needed to be present - made no difference upon testing.

![2023-05-30_18-09-35](https://github.com/Kotlin/dokka/assets/22685910/c02ae323-33a6-4b3a-b58c-df0ed3fa08e5)